guake/simplegladeapp.py

```diff
@@ -30,7 +30,7 @@
 
 class SimpleGladeApp():
 
-    def __init__(self, path, root=None, domain=None, **kwargs):
```

Comment:
Hehe, you should not taint your class for unit test.
You need to use mock and patch for that